python/fake_data/raw_udd/test.tsv

```diff
@@ -0,0 +1,3 @@
+0|1|9
```

Comment:
if the fake data is used for testing, maybe move them to the test folder.

Reply:
OK